eth/filters/bor_filter.go

```diff
@@ -76,13 +75,9 @@ func newBorBlockLogsFilter(backend Backend, sprint uint64, addresses []common.Ad
 func (f *BorBlockLogsFilter) Logs(ctx context.Context) ([]*types.Log, error) {
 	// If we're doing singleton block filtering, execute and return
 	if f.block != (common.Hash{}) {
-		receipt, err := f.backend.GetBorBlockReceipt(ctx, f.block)
-		if err != nil {
-			return nil, err
-		}
-
+		receipt, _ := f.backend.GetBorBlockReceipt(ctx, f.block)
 		if receipt == nil {
```

Comment:
Why aren't we taking `error` under consideration ? Is it due to how the called function returns its value ?

Reply:
Because most of the time, bor block receipt will be `nil` when there will be no logs for bor state sync. And it's only one type - `not found`

Reply:
Oh ok 👍